src/mixedPath.ml

```diff
@@ -11,6 +11,13 @@ type t =
 let of_name (name : Name.t) : t =
   PathName (PathName.of_name [] name)
 
+let to_string : t -> string = function
```

Comment:
I think we can put this function directly where it is used, or give it a different name, as this could be mistaken for a safe function while it does not cover all the cases.